node/src/actors/chain_manager/handlers.rs

```diff
@@ -886,6 +886,11 @@ impl Handler<PeersBeacons> for ChainManager {
             peers_beacons.ignore_no_consensus_beacons(last_superblock_consensus);
         }
 
+        if self.current_epoch.is_none() {
+            return Err(());
+        }
+        let current_epoch = self.current_epoch.unwrap();
+
         // Calculate the consensus, or None if there is no consensus
         let consensus_threshold = self.consensus_c as usize;
         let beacon_consensus = peers_beacons.superblock_consensus(consensus_threshold);
```

Comment:
I know that it is not parat of this PR, but in the function `superblock_consensus` does not describe in the code what is the `bool` :)

Reply:
It's called `is_there_block_consensus`, returns true if the most common superblock has more than `consensus_threshold` % of consensus.

Reply:
most common superblock and most common block